### ISSUE TITLE
importcontent command should allow failure on import errors

### ISSUE THREAD

**Issue by dbnicholson** [learningequality/kolibri#9258]
## Observed behavior

When running the `importcontent` management command, it succeeds with a warning when files failed to be imported.

## Errors and logs

```
WARNING  32 files are skipped, because errors occurred during the import.
```

## Expected behavior

`importcontent` fails when it can't import all files.

## User-facing consequences

If you're trying to provision a fully populated device, then you'll have to dig through the logs to find out if it really succeeded or not.

## Steps to reproduce

```
python -m kolibri manage importchannel network 2091ca47ff544c96b4ae02b3a92346e1
python -m kolibri manage importcontent network --baseurl https://example.com 2091ca47ff544c96b4ae02b3a92346e1
echo $?
```

The last step would print `0` while I'd expect it to have some non-`0` failure code.

## Context

This is the develop branch as of 6f7f0b866e71fd4e89ff9bf9a3a686738bee29af on Linux (EndlessOS). Where this really affects us is in our image builder. There we're installing Kolibri via the `org.learningequality.Kolibri` flatpak app in flathub. Currently this uses kolibri-installer-gnome 2.0, which in turn includes Kolibri 0.14.7 (AFAICT).

We want to pre-populate our OS images with Kolibri content. If the content isn't reliably downloaded, then we might ship images that are missing channel objects.

**Comment by rtibbles** [comment]
This is deliberate, as much of the download is taking place in a places with unreliable connectivity - and causing the entire import to fail because of a few files is undesirable.

A fix that would encompass the use case you are describing would be to add a `--fail-fast` or `--error-on-fail` flag to the command that could cause the command to fail when a file download fails.

**Comment by dbnicholson** [comment]
Sorry, I should have phrased that differently. I did find the old commits that made it that way. I'll make a PR with a flag.

**Comment by rtibbles** [comment]
Fixed in #9259
